dessia_common/forms.py

```diff
@@ -88,40 +108,66 @@ def voldmlr_primitives(self):
         return volumes
 
 
-DEF_SS = StandaloneSubobject.generate(1)
+class StandaloneBuiltinsSubobject(EmbeddedBuiltinsSubobject):
+    """
+    Overwrites EmbeddedBuiltinsObject to make it standalone.
+
+    :param distarg: A Distance with units
+    :type distarg: Distance
+    :param floatarg: A float
+    :type floatarg: float
+    :param intarg: An integer
+    :type intarg: int
+    :param boolarg: A boolean
+    :type boolarg: bool
+    :param name: Object's name
+    :type name: str
+    """
+    _standalone_in_db = True
 
+    def __init__(self, distarg: Distance, floatarg: float, intarg: int,
+                 boolarg: bool, name: str = 'Standalone Subobject'):
 
-class EnhancedStandaloneSubobject(StandaloneSubobject):
-    def __init__(self, floatarg: Distance, boolarg: bool,
-                 name: str = 'Standalone Subobject'):
-        self.boolarg = boolarg
+        EmbeddedBuiltinsSubobject.__init__(self, distarg=distarg, floatarg=floatarg,
+                                           intarg=intarg, boolarg=boolarg, name=name)
+
+
+DEF_EBS = EmbeddedBuiltinsSubobject.generate(1)
+DEF_SBS = StandaloneBuiltinsSubobject.generate(1)
 
-        StandaloneSubobject.__init__(self, floatarg=floatarg, name=name)
+
+class EnhancedStandaloneSubobject(StandaloneBuiltinsSubobject):
+    def __init__(self, floatarg: Distance, name: str = 'Standalone Subobject'):
+        StandaloneBuiltinsSubobject.__init__(self, distarg=floatarg, floatarg=floatarg, intarg=floor(floatarg),
+                                             boolarg=floatarg.is_integer(), name=name)
 
     @classmethod
     def generate(cls, seed: int) -> 'EnhancedStandaloneSubobject':
         floatarg = Distance(1.2 * seed)
-        boolarg = floatarg.is_integer()
         name = 'EnhancedStandaloneSubobject' + str(seed)
-        return cls(floatarg=floatarg, boolarg=boolarg, name=name)
+        return cls(floatarg=floatarg, name=name)
 
 
 DEF_ESS = EnhancedStandaloneSubobject.generate(1)
 
 
-class InheritingStandaloneSubobject(StandaloneSubobject):
-    def __init__(self, floatarg: Distance, strarg: str,
+class InheritingStandaloneSubobject(StandaloneBuiltinsSubobject):
+    def __init__(self, distarg: Distance, floatarg: float, intarg: int, boolarg: bool, strarg: str,
                  name: str = 'Inheriting Standalone Subobject'):
         self.strarg = strarg
 
-        StandaloneSubobject.__init__(self, floatarg=floatarg, name=name)
+        StandaloneBuiltinsSubobject.__init__(self, distarg=distarg, floatarg=floatarg, intarg=intarg,
+                                             boolarg=boolarg, name=name)
 
     @classmethod
     def generate(cls, seed: int) -> 'InheritingStandaloneSubobject':
-        floatarg = Distance(0.7 * seed)
+        distarg = Distance(0.7 * seed)
+        floatarg = 0.1 * seed
         strarg = str(-seed)
```

Comment:
str(-seed) ? Why not str(seed) ?

Reply:
Here, it serves the only purpose on testing frontend and platform behavior.